src/main/sensors/battery.c

```diff
@@ -485,6 +614,26 @@ uint16_t getBatteryVoltageLatest(void)
     return voltageMeter.unfiltered;
 }
 
+uint16_t getOptionBatteryVoltage(void) /// wat to call osd ant telemitry
```

Comment:
This is not done properly - this will mean that depending on whether a delay is used, either filtered or unfiltered voltage might be used. Also, using unfiltered voltage for this seems to be just plain wrong.